modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -655,15 +655,6 @@ definition {
 			locator1 = "MyWorkflowTasks#ASSIGNED_TO_MY_ROLES_TABLE_ASSET_TITLE");
 	}
 
-	@description = "Verify that in Update Due Date modal, the user can expand the comment box"
```

Comment:
These tests were either discarded or being covered by a different test. I removed them from the test cases.

Reply:
[LRQA-68860](https://liferay.atlassian.net/browse/LRQA-68860) This test was removed from the test map